app/locales/en/translations.js

```diff
@@ -294,6 +294,7 @@ export default {
     'month': 'Month',
     'today': 'Today',
     'backToCalendar': 'Back to Calendar',
+    'loadingEvents': 'Loading Events...',
```

Comment:
I think for the french it will be best to use `Rassembler les événements...` ("getting events...")